cli/waiter/action.py

```diff
@@ -287,3 +287,70 @@ def process_ping_request(clusters, token_name_or_service_id, is_service_id, time
                 success = True
         overall_success = overall_success and success
     return overall_success
+
+
+def send_signal_to_instance_on_cluster(cluster, signal_type, service_id, instance_id, timeout_seconds):
+    """Send sigkill request to the specific instance"""
+    cluster_name = cluster['name']
+    http_util.set_retries(0)
+    try:
+        print(f'Sending {terminal.bold(signal_type)} request to instance {terminal.bold(instance_id)} in {terminal.bold(cluster_name)}...')
+        params = {'timeout': timeout_seconds * 1000, 'service-id' : service_id}
+        # DELETE REQ
+        #resp = http_util.delete(cluster, f'/apps/{instance_id}/{signal_type}', params=params, read_timeout=30) 
+        resp = http_util.post(cluster, f'/apps/{instance_id}/{signal_type}', '', params=params)
```

Comment:
Please use a format like:
```suggestion
        resp = http_util.post(cluster, f'/apps/{service_id}/signal/{signal_type}/{instance_id}', '', params=params)
```

`apps` refers to services, so it is strange to make the path after `/apps` an instance ID.
the signal and instance ID order matches the order in the CLI.